npm/private/npm_package_store.bzl

```diff
@@ -238,6 +238,7 @@ def _npm_package_store_impl(ctx):
                         "--directory",
                         package_store_directory.path,
                     ],
+                    env = {"LC_ALL": "en_US.UTF-8"},
```

Comment:
Does every platform in the world have `en_US`? 😅

Reply:
I think this can be anything ending in UTF-8 actually. I've seen `C.UTF-8` too. Lemme research what's the most correct value.

Reply:
`rules_js.UTF-8`? 😄

Reply:
The libarchive tests definitely handle cases where `en_US.UTF-8` isn't on the machine. I'm not a big fan of trying to invent something here - and I think it's likely we're just fixing one specific case of a general problem. Users may be better off fixing the locale of their Bazel environment since this isn't the only spot that can go wrong.

Reply:
As long as you looked into it 👍

Reply:
https://github.com/bazelbuild/rules_java/blob/767e4410850453a10ccf89aa1cededf9de05c72e/toolchains/utf8_environment.bzl has a well-reasoned answer